crates/driver/src/infra/api/routes/settle/dto/solution.rs

```diff
@@ -8,4 +8,7 @@ pub struct Solution {
     /// Unique ID of the solution (per driver competition), to settle.
     #[serde_as(as = "serde_with::DisplayFromStr")]
     solution_id: u64,
+    #[allow(dead_code)]
+    /// The last block number in which the solution TX can be included
+    pub block_deadline: u64,
```

Comment:
I assume we do not need to act on this on our driver (the same as with the `solution_id`.

Reply:
The driver already has a submission deadline in a config file. It should now be read from the request and the config argument should be dropped.